index.html

```diff
@@ -160,6 +171,11 @@ <h2 class="text-danger">Dinner</h2>
           </div>
         </div>
       </div>
+      <div class="whole-program">
+        <p class="text-white text-center">
+          <u><a class="text-white" href="#">See the whole program</a></u>
+        </p>
+      </div>
```

Comment:
- [ ] Good job adding the missing link,but I shouldn't show up on the mobile version, kindly fix that



![image](https://user-images.githubusercontent.com/89282221/190502198-a99491e3-df56-4f85-9cb0-dc05acdcf457.png)